qemu/tests/cfg/balloon_memhp.cfg

```diff
@@ -0,0 +1,41 @@
+# Notes:
+#   Need add movable_node in kernel line for rhel8 guest, rhel7 won't work well
+#   with memory unplug.
+- balloon_memhp:
+    type = balloon_memhp
+    slots_mem = 4
+    size_mem = 1G
+    maxmem_mem = 32G
```

Comment:
Is 32G big enough? 
Sometimes we reserve host mem >64G, then guest boot mem is 32G already.
Is there any bad effect if set this value larger?

Reply:
No, it can be larger. How about 64G? BTW, If we run the test on a very big machine, I guess we'd better limit the memory to a fixed value, otherwise any value for maxmem would fail when hit bigger machine. What do you think?